shardingsphere-kernel/shardingsphere-sql-translator/shardingsphere-sql-translator-core/pom.xml

```diff
@@ -33,5 +33,10 @@
             <artifactId>shardingsphere-sql-translator-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shardingsphere</groupId>
+            <artifactId>shardingsphere-sql-translator-native-provider</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Core module should not aware the provider modules

Reply:
move this import to shardingsphere-infra-rewrite module? @terrymanu